container-search/src/main/java/com/yahoo/search/logging/AbstractSpoolingLogger.java

```diff
@@ -31,8 +32,11 @@ public AbstractSpoolingLogger() {
     public AbstractSpoolingLogger(Spooler spooler) {
         this.spooler = spooler;
         this.executorService = new ScheduledThreadPoolExecutor(1, new DaemonThreadFactory("AbstractSpoolingLogger-send-"));
-        // Delay some time before starting to process files, subclasses might need to be constructed fully first
-        this.executorService.scheduleWithFixedDelay(this, 5, 1L, TimeUnit.SECONDS);
+    }
+
+    /** Start processing files, must be called by subclasses */
+    public void start() {
```

Comment:
Our implementation/subclass calls start() at end of constructor now, if anyone else subclass this they need to call start() explicitly, but that is they way we want to do it, so that should not be necessary.